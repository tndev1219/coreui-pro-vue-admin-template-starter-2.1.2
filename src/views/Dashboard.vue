<template>
  <div class="animated fadeIn">
    <div id="Dashboard">Dashboard</div>
  </div>
</template>

<script>
export default {
  name: 'dashboard'
}
</script>
