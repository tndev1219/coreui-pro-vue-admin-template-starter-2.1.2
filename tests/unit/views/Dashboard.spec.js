import Vue from 'vue'
import { shallowMount } from '@vue/test-utils'
import BootstrapVue from 'bootstrap-vue'
import Dashboard from '@/views/Dashboard'

Vue.use(BootstrapVue)

describe('Dashboard.vue', () => {
  it('has a name', () => {
    expect(Dashboard.name).toMatch('dashboard')
  })
  it('is Vue instance', () => {
    const wrapper = shallowMount(Dashboard)
    expect(wrapper.isVueInstance()).toBe(true)
  })
  it('is Dashboard', () => {
    const wrapper = shallowMount(Dashboard)
    expect(wrapper.is(Dashboard)).toBe(true)
  })
  it('should render correct content', () => {
    const wrapper = shallowMount(Dashboard)
    expect(wrapper.find('#Dashboard').text()).toMatch('Dashboard')
  })
  test('renders correctly', () => {
    const wrapper = shallowMount(Dashboard)
    expect(wrapper.element).toMatchSnapshot()
  })
})
