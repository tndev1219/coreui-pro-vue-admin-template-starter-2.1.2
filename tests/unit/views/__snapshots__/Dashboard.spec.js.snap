// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard.vue renders correctly 1`] = `
<div
  class="animated fadeIn"
>
  <div
    id="Dashboard"
  >
    Dashboard
  </div>
</div>
`;
